src/index.scss

```diff
@@ -0,0 +1,231 @@
+      &.is-active {
+        transition: $hover-transition;
+        color: $primary-color;
+      }
+
+      &:hover::after,
+      .nav__item:first-child &::after {
+        transition: $hover-transition;
+        display: block;
+      }
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid $secondary-color;
+  border-radius: $border-radius;
+  transition: $hover-transition;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-image: url(./images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: $spacing-large $spacing-medium;
+  }
+
+  &__title {
+    font-family: $font-family-roboto;
+    font-size: $font-size-medium;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $text-color-dark;
+    width: 166px;
+    height: 36px;
+    margin-left: $spacing-medium;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: $font-family-roboto;
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $text-color-light;
+    margin-left: $spacing-medium;
+    margin-bottom: 15px;
+  }
+
+  &__reviews-wrapper {
+    display: flex;
+    width: 166px;
+    margin-left: $spacing-medium;
+    margin-bottom: 20px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: $font-family-roboto;
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price-info {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: $spacing-medium;
+    margin-bottom: 20px;
+  }
+
+  &__text {
+    font-family: $font-family-roboto;
+    font-size: $font-size-medium;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__price {
+    font-family: $font-family-roboto;
+    font-size: $font-size-large;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    margin-bottom: $spacing-medium;
+    margin-left: $spacing-medium;
+    background-color: $primary-color;
+    border-radius: $border-radius;
+
+    &:hover {
+      background-color: $white;
+      border: 1px solid $primary-color;
+    }
+
+    &-text {
+      font-family: $font-family-roboto;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-align: center;
+      color: $white;
+
+      &:hover {
+        color: $primary-color;
+      }
+    }
+  }
+}
+
+.star {
+  display: inline-block;
+  background-image: url(./images/star.svg);
+  width: $star-width;
+  height: $star-height;
+  border-radius: 0.5px;
+
+  &--active {
+    background-image: url(./images/star-active.svg);
+  }
```

Comment:
The class `&--active` under `.star` should be reviewed for BEM consistency. Ensure that this modifier is used consistently across the HTML and CSS files.